### ISSUE TITLE
[wg/ag] Alignment on Overall Architecture for WCAG-3 - AGWG charter

### ISSUE THREAD

**Issue by sydneycolemanfromgoogle** [w3c/charter-drafts#789]
Alignment on Overall Architecture for WCAG-3: Right now, we cannot explain what WCAG-3 will look like to accessibility experts at our company. We need to be able to agree upon the overall plan with the key pieces outlined: what is and is not in scope for WCAG-3.0 and how WCAG-3 approaches  third party content, and so forth. So far, AGWG has focused on the assertion  details without alignment on the overall structure, and we believe this approach is ineffective.

**Comment by sydneycolemanfromgoogle** [comment]
@jyasskin

**Comment by iadawn** [comment]
The overall scope for WCAG 3 has been laid down and refined in a [number of working drafts](https://www.w3.org/standards/history/wcag-3.0/) over the last few years. The work of the group over the next charter is to continue to refine this. A key part of this work is to continue to specify what is and is not in scope for WCAG 3.

AGWG work over the last two years has covered significantly more than 'assersion details'. The work of the sub-groups has extensively scrutinzed and developed requirements and assertions that form the current draft for WCAG 3. This is built on a framework that was [explored extensively by the group](https://www.w3.org/2024/02/06-ag-minutes.html#t01) in early 2024.

**Comment by sydneycolemanfromgoogle** [comment]
Do you believe we have landed on a strong working draft for the next charter to refine?

**Comment by iadawn** [comment]
I accept that the consensus of the group was in support of the current published working draft.
